service/controller/v2/resource/drainer/evict.go

```diff
@@ -0,0 +1,34 @@
+package drainer
+
+import (
+	"github.com/giantswarm/microerror"
+	"k8s.io/api/core/v1"
+	"k8s.io/api/policy/v1beta1"
+	apismetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+// evict pod from node
+func EvictPod(k8sClient kubernetes.Interface, pod v1.Pod) error {
+	var deleteGracePeriod int64 = 60
```

Comment:
I need a refresher here. How does the grace period behave with the client? Will the eviction be scheduled and the k8s client returns immediately or will the client when calling `Evict` block for a minute now? When it is the first case, this is fine.

Reply:
it will returns immediately, it does not block